internal/pkg/aws/codepipeline/codepipeline.go

```diff
@@ -34,6 +35,12 @@ type Pipeline struct {
 	Name string `json:"name"`
 }
 
+// PipelineStatus represents a Pipeline's status.
+type PipelineState struct {
+	PipelineName string `json:"Pipeline"`
```

Comment:
probably want this to be just `pipelineName`, so as to not get this confused with an actual pipeline object